src/sessions/softphone-session-handler.ts

```diff
@@ -100,6 +102,18 @@ export default class SoftphoneSessionHandler extends BaseSessionHandler {
     this.handleSoftphoneConversationUpdate(update, participant, callState, session);
   }
 
+  holdOtherSessions(currentSession: IExtendedMediaSession): void {
+    const sessions = this.sessionManager.getAllActiveSessions();
+    // Hold only softphone sessions and sessions not currently held.
+    const otherSessions = sessions.filter(session => session.sessionType === SessionTypes.softphone && !session.pcParticipant?.calls[0].held && session !== currentSession);
```

Comment:
I adjusted this to filter out anything not a softphone session and already held. I'm curious though, is there a better way to check if a session is currently held? I had to dig pretty deep on the session to find this @jensengar